services/jenkins-pipeline-monitor/deploy_lambda.sh

```diff
@@ -0,0 +1,16 @@
+set -e
+
+echo "Deployment stage (test, prod)"
+read config_dir
```

Comment:
looks like `config_dir` tells if it is test or prod. where is this set?

Reply:
read command reads for the user input